VideoLocker/src/main/java/org/edx/mobile/util/images/TopAnchorWidthFillTransformation.java

```diff
@@ -0,0 +1,49 @@
+package org.edx.mobile.util.images;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.os.Build;
+
+import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool;
+import com.bumptech.glide.load.resource.bitmap.BitmapTransformation;
+
+/**
+ * Created by miankhalid on 9/9/15.
+ */
+public class TopAnchorWidthFillTransformation extends BitmapTransformation {
```

Comment:
I think this can be renamed to `TopAnchorFillWidthTransformation` for slightly easier comprehension.